src/main/webapp/dev/js/main/feedbackSubmissionsEdit.js

```diff
@@ -153,6 +153,47 @@ function focusModeratedQuestion() {
     }
 }
 
+function hideFeedbackParticipantCommentOption(indexSuffix) {
+    if ($(`#responseCommentRow${indexSuffix}`).length) {
+        $(`#responseCommentTable${indexSuffix}`).closest('div').hide();
+    } else if ($(`#showResponseCommentAddForm${indexSuffix}`).is(':visible')) {
+        $(`#responseCommentTable${indexSuffix}`).closest('div').hide();
+    } else {
+        $(`#button_add_comment${indexSuffix}`).closest('div').hide();
+    }
+}
+
+function showFeedbackParticipantCommentOption(indexSuffix) {
+    if ($(`#responseCommentRow${indexSuffix}`).length) {
+        $(`#responseCommentTable${indexSuffix}`).closest('div').show();
+    } else if ($(`#showResponseCommentAddForm${indexSuffix}`).is(':visible')) {
+        // do nothing
+    } else {
+        $(`#button_add_comment${indexSuffix}`).closest('div').show();
+    }
+}
+
+function checkResponsesForFeedbackParticipantComment(indexSuffix) {
+    const radioButton = $(`input[name='responsetext${indexSuffix}']:checked`);
+
+    if (!radioButton.length) {
+        hideFeedbackParticipantCommentOption(indexSuffix);
+    } else if (radioButton.data('text') === 'otherOptionText'
+        && $(`#otherOptionText${indexSuffix}`).val().length === 0) {
+        hideFeedbackParticipantCommentOption(indexSuffix);
+
+        $(`#otherOptionText${indexSuffix}`).keyup(function () {
```

Comment:
This line is giving me lint error.   `185:51  error  Unexpected function expression  prefer-arrow-callback`.
I am not sure what I am doing wrong.

Reply:
@LuLechuan Can you help me with this?